server/src/perl/lib_bs22/pltags.pm

```diff
@@ -324,14 +325,32 @@ sub build_pltags {
         }
 
         elsif (($sActiveOO->{"Dancer"} or $sActiveOO->{"Dancer2"} or $sActiveOO->{"Mojolicious::Lite"}) and
-            $stmt=~/^(?:get|any|post|put|delete)\s+(?:[\s\w,\[\]'"]+=>\h*)?(['"])(\/[\w\/:\-]*)\1\s*=>\s*sub/) { # Routing paths
-            my $route = $2;
+            $stmt=~/^(?:any|before\_route)\s+\[([^\]]+)\]\s+(?:=>\h*)?(['"])([^\2]+)\2\s*=>\s*sub/) { # Multiple request routing paths
+            my $requests = $1;
+            if ($requests) {
```

Comment:
I believe `$requests` will always be truthy because of the + in `([^\]]+)`

Reply:
Agreed, removed the if.